service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateStorageAuditLogSettingsStep.java

```diff
@@ -46,6 +49,15 @@ protected void createResource(FlightContext context, ArmManagers armManagers) {
             .withLog("StorageWrite", 0)
             .withLog("StorageDelete", 0)
             .create();
+
+    context
+        .getWorkingMap()
+        .put(
+            STORAGE_AUDIT_LOG_SETTINGS_KEY,
```

Comment:
@aherbst-broad do we consider using it later somehow?

Reply:
I'm setting it here so we can get at the expected diagnostic settings in the associated test.